### PR TITLE
Adding /metrics endpoint to the integrations

### PR DESCRIPTION
**Pull request by apahim**
By replacing the BASH script by a Python equivalent, we can add the /metrics
endpoint and report relevant information:

    qontract_reconcile_last_run_seconds{integration="quay-mirror"} 73.32808894099435
    qontract_reconcile_run_status_total{integration="quay-mirror",status="0"} 5.0
    qontract_reconcile_run_status_total{integration="quay-mirror",status="1"} 1.0

### CODE REVIEW

File: dockerfiles/hack/run_integration.py [hunk 1]
```diff
@@ -0,0 +1,91 @@
...
+            return process.poll()
+
+
+if __name__ == "__main__":
+    start_http_server(8080)
+
+    run_time_gauge = Gauge(name='qontract_reconcile_last_run_seconds',
+                           documentation='Last run duration in seconds',
+                           labelnames=['integration_name'])
+
```
[the remark this answers is not in the document]
Comment (apahim) [reply dockerfiles/hack/run_integration.py:78]: Done

**Comment by jmelis** [comment]
I just realized we probably need a label for shards and shard_id so we have separare stats per sharded integrarion.

**Comment by jmelis** [comment]
Not in this PR, we should do it in the shards PR, I think.

File: dockerfiles/hack/run_integration.py [hunk 2]
```diff
@@ -0,0 +1,91 @@
...
+    if DRY_RUN is not None:
+        cmd.append(DRY_RUN)
+
+    cmd.append(INTEGRATION_NAME)
+
+    if INTEGRATION_EXTRA_ARGS is not None:
+        cmd.extend(INTEGRATION_EXTRA_ARGS.split())
+
+    process = subprocess.Popen(cmd, stdout=subprocess.PIPE,
```
[the remark this answers is not in the document]
Comment (jmelis) [reply dockerfiles/hack/run_integration.py:51]: I see, I missed that. LGTM!

**Pull request merged**